learn/get-started/build-a-data-service-in-ballerina.md

```diff
@@ -205,6 +212,9 @@ The `mysql:Client` provides two primary remote methods for performing queries.
 To use the `query()`, `queryRow()`, and `execute()` methods, which can perform basic CRUD operations against the MySQL database, add the code below to the `main.bal` file. 
 
 ```ballerina
+import ballerina/sql;
```

Comment:
```suggestion
import ballerina/sql;
```
Don't we need to remove this unused import?

Reply:
The unused import was in a previous sample. It has been removed from there.

https://github.com/ballerina-platform/ballerina-dev-website/pull/5037/files#diff-439f92e0c6479c601f714f2743e690239e2745cf2eaecd4de77817f7c9ad9836L169

Reply:
Fixed in [acdbdda](https://github.com/ballerina-platform/ballerina-dev-website/pull/5037/commits/acdbdda187290416c649754ed6c66fceeefc2dca)